api/vendor/github.com/openshift/hive/apis/hive/v1/clusterrelocate_types.go

```diff
@@ -6,9 +6,9 @@ import (
 
 // ClusterRelocateSpec defines the relocation of clusters from one Hive instance to another.
 type ClusterRelocateSpec struct {
-	// KubeconfigSecretRef is a reference to the secret containing the kubeconfig for the destination Hive instance.
+	// RemoteKubeconfigRef is a reference to the secret containing the kubeconfig for the destination Hive instance.
```

Comment:
Why is this being changed? Maybe just a find/replace gone wrong?

Reply:
Good catch. Removing the redundant change